easybuild/easyconfigs/i/icc/icc-2016.1.150-GCC-4.9.3.eb

```diff
@@ -8,7 +8,7 @@ description = "C and C++ compiler from Intel"
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
-sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_%(version)s.tgz']
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_update%(version_minor)s.tgz']
```

Comment:
no, EB will require both sources

in the 2016.0.109 easyconfigs, we'll need to use `parallel_studio_xe_%(version_major)s_composer_edition_for_cpp.tgz` indeed (see https://github.com/DirkdeDraak/easybuild-easyconfigs/pull/1)